.github/workflows/examples.yaml

```diff
@@ -46,11 +38,11 @@ jobs:
         uses: ARM-software/cmsis-actions/vcpkg@main
         with:
           config: ./CMSIS/RTOS2/FreeRTOS/Examples/vcpkg-configuration.json
-          
+
       - name: Activate Arm tool license
```

Comment:
We could update to use [ARM-software/cmsis-actions/armlm](https://github.com/ARM-software/cmsis-actions/blob/main/README.md#action-armlm)